### ISSUE TITLE
How can I make linter to fail, if there will be any filtered findings

### ISSUE THREAD

**Issue by artur-krysiak** [reviewdog/action-actionlint#47]
I use this action to lint written workflows and found `filtered finding` indication useful. Is there any way to make the linter fail, when there will be any filtered finding?

**Comment by shogo82148** [comment]
Sorry for the long silence on this one.

reviewdog's `-fail-level` only evaluates findings that pass the filter (i.e. the ones that actually get reported as comments/checks). Findings that get dropped by `-filter-mode` (e.g. because they're outside the diff) are never considered for the fail decision, so there's no dedicated "fail if anything was filtered out" option.

However, you can get the effect you want by setting `filter_mode: nofilter` together with `fail_level`. With `nofilter`, nothing gets filtered out in the first place — every actionlint finding (not just the ones on changed lines) is reported and counted toward `fail_level`:

```yaml
- uses: reviewdog/action-actionlint@v1
  with:
    filter_mode: nofilter
    fail_level: error   # or "any"
```

Note that this changes the behavior from "only lint the diff" to "lint and report the whole file/repo", so you'll start seeing (and failing on) pre-existing issues outside your changes too — which may or may not be what you want, but it does ensure nothing is silently filtered away.

Both `filter_mode` and `fail_level` are already supported inputs of this action, so no code changes are needed here.
